services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -44,17 +47,28 @@ export const DropdownField = ({
   const formatOptions = (options: DropdownOptions[] | string) => {
     let dropdownOptions = [];
     if (typeof options === "string") {
-      const dynamicOptionValues = report?.fieldData[options];
-      if (dynamicOptionValues) {
-        const fieldOptions = dynamicOptionValues.map((option: EntityShape) => ({
+      const dynamicOptionValues =
+        options === "copyEligibleReports"
+          ? submittedReportsByState
+          : report?.fieldData[options];
+      if (options === "copyEligibleReports" && dynamicOptionValues) {
+        dropdownOptions = dynamicOptionValues.map(
+          (option: ReportMetadataShape) => ({
+            label:
+              option.programName + " " + convertDateUtcToEt(option.dueDate),
+            value: option.fieldDataId,
+          })
+        );
+      } else if (dynamicOptionValues) {
+        dropdownOptions = dynamicOptionValues.map((option: EntityShape) => ({
           label: option.name,
           value: option.id,
         }));
-        dropdownOptions = fieldOptions;
       }
     } else {
       dropdownOptions = options;
     }
```

Comment:
We may be able to reduce CodeClimate's judgement of the cognitive complexity here by replacing this section with this code. I _believe_ it's equivalent, and has fewer `if` statements:

```ts
    let dropdownOptions = [];
    if (options === "copyEligibleReports") {
      dropdownOptions = submittedReportsByState?.map(option => ({
          label: `${option.programName} ${convertDateUtcToEt(option.dueDate)}`,
          value: option.fieldDataId!,
        })
      ) ?? [];
    }
    else if (typeof options === "string") {
      dropdownOptions = report?.fieldData[options]?.map((option: EntityShape) => ({
        label: option.name,
        value: option.id,
      })) ?? [];
    } else {
      dropdownOptions = options;
    }
```

On the other hand, it may be worse! If it's possible for us to run Code Climate checks locally, I haven't figured out how.